src/lib/pull-request.coffee

```diff
@@ -25,6 +25,15 @@ pr = (token, branch, body, cb) ->
   If I do something wrong, [blame a human](https://github.com/cfpb/hubot-capital-framework/issues/new).
 
   ![kitten gif](http://thecatapi.com/api/images/get?format=src&type=gif)
+
+  ## You are not
+
+  After this PR is merged, you should be prepared to update the CF dependencies I've released in the following projects:
+
+  - [ ] [capital-framework gh-pages branch](https://github.com/cfpb/capital-framework/tree/gh-pages)
+  - [ ] [design-manual](https://github.com/cfpb/design-manual/)
+  - [ ] [generator-cf](https://github.com/cfpb/generator-cf/)
```

Comment:
I'd update this to make it clear that what needs updating is the template for `package.json` which provides the CF dependencies in the generated project, and the template example page.

Reply:
It's more than that, it's possible a release could require updating anything in the template directory, like I just found with the icons change.

Reply:
True. I guess the thing I want to make clear is that you're updating the template. CF isn't itself a dependency of the generator-cf package.